src/config/index.ts

```diff
@@ -0,0 +1,9 @@
+const config = {
+  DATABASE_URL: process.env.DATABASE_URL_DEV,
+};
+
+if (process.env.NODE_ENV === 'development') {
+  config.DATABASE_URL = process.env.DATABASE_URL_TEST;
+}
```

Comment:
you can change it to 
```javascript
DATABASE_URL: process.env.NODE_ENV === 'development': 
process.env.DATABASE_URL_TEST ? process.env.DATABASE_URL_DEV
```

then place it on `main.ts` on 25 line that is setting up the mongo database url

Reply:
i don't know how to do it

Reply:
fixed